doc/chapters/gateways/10-cgroups-gateway.rst

```diff
@@ -18,11 +18,23 @@ with the value to apply.
 No syntax or other checks for correctness is performed on the key/value pairs,
 see the `lxc.container.conf` man page for more details about valid settings.
 
-Before appying the configuration it is advised to check if the cgroup option is enabled in
-the system's kernel. Following command will list available cgroup options::
+Kernel Options
+--------------
+To be able to limit memory usage with cgroups, ``CONFIG_CGROUPS``, ``CONFIG_MEMCG`` and
+``CONFIG_MEMCG_SWAP`` should be enabled. And also before appying the configuration it is
```

Comment:
I would prefer to not start the sentence with "And", so I would remove "And also" (so the sentence starts with "Before applying" (also typo in applying).